src/scripts/main.js

```diff
@@ -1,7 +1,75 @@
-'use strict';
+const Game = require('../modules/Game.class');
+
+// eslint-disable-next-line no-unused-expressions
+('use strict');
 
 // Uncomment the next lines to use your game instance in the browser
 // const Game = require('../modules/Game.class');
```

Comment:
This commented out line of code seems to be unnecessary since the Game class is already imported at the beginning of the file. It's a good practice to remove commented out code to keep your codebase clean.